docs/contributing/introduction-to-contributing.md

```diff
@@ -16,66 +16,29 @@ please read the [code of conduct](./code-of-conduct.md) and follow the direction
 3. Always review your code first. Run the project locally and test it before requesting a review.
 4. Always communicate in the GitHub repository. Whether in the issue or the pull request, keeping the lines of communication open and visible to everyone on the team helps everyone around you.
 
-## Setup
-
-1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository you're working on from [github/open-sauced](https://github.com/open-sauced) to your own GitHub account.
-2. Clone the forked repository to your local machine.
-3. Check the project's README to determine how to run the project locally.
-
-For many of our repositories, you'll find the directions below work for running the project locally:
-
-Run `npm ci` to install the dependencies and set up the project.
-
-You can also use the shell commands below to get started once you have forked the repository:
-
-```shell
-git clone https://github.com/<GITHUB_USERNAME>/<REPO_NAME>.git
-cd <REPO_NAME>
-npm ci
-```
-
-Make sure to replace `<GITHUB_USERNAME>` with your GitHub username.
-
-## Building
-
-To generate a production-ready version of your code, run:
-
-```shell
-npm run build
-```
-
-## Testing
+## Issues
 
-For running the test suite, use the following command:
+If you wish to work on an open issue, please comment on the issue with `.take`, and it will be assigned to you. If an issue is not assigned, it is assumed to be available for anyone to work on. Please assign yourself to an issue before working on it to avoid conflicts.
 
-```shell
-# the tests will run in watch mode by default
-npm test
-```
+If you contribute to the project for the first time, you can search for issues with `good first issue` or `bug` labels in the repository.
 
-Since the tests run in watch mode by default, some users may encounter errors about too many files being open. In this case, it may be beneficial to [install Watchman](https://facebook.github.io/watchman/docs/install.html).
+In case you get stuck, feel free to ask for help in the [Discord](https://discord.gg/U2peSNf23P) server or [GitHub Discussions](https://github.com/orgs/open-sauced/discussions).
 
-Check out this course, [Testing JavaScript](https://testingjavascript.com/), for more info on testing React and JavaScript.
+Please follow our [code of conduct](./code-of-conduct.md) in all your interactions with the project and its contributors.
 
-## Applying Lint Styleguide
+## Commits
```

Comment:
Also, I'm debating between placing this section here or after the Pull Request section.